src/Formatter/HumanReadableExceptionFormatter.php

```diff
@@ -23,25 +22,47 @@ public function __invoke(array $options)
     public function format(array $record): string
     {
         $exception = $record['context']['exception'] ?? null;
-        if ($exception) {
-            return $this->printFromException($exception);
-        } else {
-            return $this->printWithoutException($record);
+
+        if (!$exception) {
+            return parent::format($record);
         }
+
+        return $this->printFromThrowable($record, $exception);
     }
 
-    protected function printWithoutException(array $record): string
+    protected function printFromThrowable(array $record, \Throwable $throwable)
     {
-        return sprintf("[%s] %s: %s\n", ...[
-            date('r'),
+        $record = $this->normalize($record);
+
+        $result = sprintf(
+            "[%s] %s.%s: %s\n\n",
+            $record['datetime'],
+            $record['channel'],
             $record['level_name'],
             $record['message']
-        ]);
+        );
+
+        $result .= "[Context]\n";
+        $result .= sprintf("  Type: %s\n", get_class($throwable));
+        $result .= sprintf("  Code: %d\n", $throwable->getCode());
+        $result .= sprintf("  File: %s\n", $throwable->getFile());
+        $result .= sprintf("  Line: %d\n\n", $throwable->getLine());
+
+        $result .= "[Trace]\n";
+        $result .= $this->buildTraceOutput($throwable->getTraceAsString());
```

Comment:
I didn't use getTraceAsString() before because I thought we can do better

- The lines it produces are pretty long, often don't fit on the screen. This is because it has both the class name and file path, but you probably don't need to see the path if you already have the class name.
- It shows no information on the function arguments
- I think we can do better in the case of nested exceptions

Reply:
Until we find a better solution I'd like to keep this one. This is better and more readable than what we had before since it comes closer to PHP's native display of errors. So it's more familiar.